src/generators/essudpgen/ReadoutGeneratorBase.cpp

```diff
@@ -7,6 +7,7 @@
 //===----------------------------------------------------------------------===//
 // GCOVR_EXCL_START
 
+#include "Error.hpp"
```

Comment:
Is this a new file?

We don't use .hpp and please remember to use <> for includes ;-)

Reply:
This is something was sneaked in by my idea. I will clean this.